llvm/test/MC/Disassembler/AMDGPU/gfx10_mubuf.txt

```diff
@@ -1,5 +1,4 @@
-# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1010 -mattr=+wavefrontsize32 -disassemble -show-encoding < %s | FileCheck -strict-whitespace -check-prefix=GFX10 %s
-# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1010 -mattr=+wavefrontsize64 -disassemble -show-encoding < %s | FileCheck -strict-whitespace -check-prefix=GFX10 %s
```

Comment:
Losing test coverage

Reply:
I think all BUF instructions are wave size independent for a given asic. So how the coverage gets lost?